USAGE.md

```diff
@@ -1144,6 +1147,20 @@ Time.now.between?("05-01".."12-01")
 Time.now.between?("5am".."11pm")
 ```
 
+#### Ephemeris
+
+{OpenHAB::CoreExt::Ephemeris Helper methods} to easily access [openHAB's Ephemeris action](https://www.openhab.org/docs/configuration/actions.html#ephemeris) are provided on all date-like objects:
+
+```ruby
+Time.now.holiday? # => false
+LocalDate.parse("12-25").holiday # => :christmas
+1.day.from_now.next_holiday # => :thanksgiving
+notify("It's #{Ephemeris.holiday_name(Date.today.holiday)}!") if Date.today.holiday?
```

Comment:
Why not `"It's #{Date.today.holiday_name}!"`

Reply:
Because at the last minute I took `holiday_name` off of the date helper methods, because I had to add extra processing to handle religious holidays, and I realized I was duplicating it for both `holiday_name` and `next_holiday_name`.

Honestly, it'd probably be best if `holiday` returned a `Holiday` object instead of a simple String or Symbol. which is how the underlying Jollyday library works, but then openHAB hides that when it abstracts it.

Reply:
If `holiday` returned a `Holiday`, then you could do `Date.today.holiday.name`
and then `Date.today == Holiday::CHRISTMAS` (or something similar) - converting a Holiday to to_zoned_date prior to comparison in the same way you do `Date.today == ZonedDateTime.now`
also `Date.today.holiday == Holiday::CHRISTMAS`

I'm just throwing things on the wall and seeing what sticks :)

Reply:
All I know is that `Ephemeris.holiday_name(Date.today.holiday)` seems painful

Reply:
Agreed. I don't want to create a Holiday class right now, but accepting something that responds to #holiday is easy enough

Reply:
`Ephemeris.holiday_name(Date.today)` works now.